drv_cfg.go

```diff
@@ -101,28 +111,28 @@ func DrvCfgQueryGUID(syscaller Syscaller) (string, error) {
 	// #nosec CWE-242, validated buffer is large enough to hold data
 	err = ioctlWrapper(syscaller, f.Fd(), opCode, &buf)
 	if err != nil {
-		return "", fmt.Errorf("QueryGUID error: %v", err)
+		return "", fmt.Errorf("queryGUID error: %v", err)
```

Comment:
Although, in general, error messages are supposed to start with a lowercase letter, this does not apply to key words, acronyms, names, etc. It's better to avoid such words at the beginning of the message where possible. Here is some explanation: https://go.dev/wiki/CodeReviewComments#error-strings

Reply:
Fixed